test/runtests.jl

```diff
@@ -32,21 +33,25 @@ end
     @test Showoff.format_fixed_scientific(Inf, 1, false) == "∞"
     @test Showoff.format_fixed_scientific(-Inf, 1, false) == "-∞"
     @test Showoff.format_fixed_scientific(NaN, 1, false) == "NaN"
-    @test Showoff.format_fixed_scientific(0.012345678, 4, true) == "12.34568×10⁻³"
-    @test Showoff.format_fixed_scientific(0.012345678, 4, false) == "1.234568×10⁻²"
-    @test Showoff.format_fixed_scientific(-10.0, 4, false) == "-1.000×10¹"
+    @test Showoff.format_fixed_scientific(0.012345678, 4, true) == "12.346×10⁻³"
+    @test Showoff.format_fixed_scientific(0.012345678, 4, false) == "1.2346×10⁻²"
+    @test Showoff.format_fixed_scientific(-10.0, 4, false) == "-1.0000×10¹"
+    @test Showoff.format_fixed_scientific(-10.0, 4, false) == "-1.0000×10¹"
+    @test Showoff.format_fixed_scientific(-10.0, 4, false)[1:end-5] == @sprintf("%0.4e", -10.0)[1:end-4]
+    @test Showoff.format_fixed_scientific(1.23456e7, 3, false)[1:end-5] == @sprintf("%0.3e", 1.23456e7)[1:end-4]
+    @test Showoff.format_fixed_scientific(2.99999999999999956E-16, 2, false) == "3.00×10⁻¹⁶"
 end
 
 @testset "Showoff" begin
     x = [1.12345, 4.5678]
     @test showoff(x) == ["1.12345", "4.56780"]
-    @test showoff([0.0, 50000.0]) == ["0", "5×10⁴"]
+    @test showoff([0.0, 50000.0]) == ["0", "5.0×10⁴"]
```

Comment:
Slightly breaking, as now engineering notation introduces the trailing 0, however it is now consistent with scientific as it also introduces the zero